recipes/pytide/build.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+set -e
+${PYTHON} setup.py build
+${PYTHON} setup.py install
```

Comment:
A pip install will do all of this and is the preferred path. Why setuptools here?

Reply:
To build the package with conda I execute these two commands that build and install the library. How to do with pip? I need the setuptools commands to build the compiled library: extension of the construction commands to interface with cmake.

Reply:
OK. But in this case, I have to declare pip as dependence in the `meta.yaml` file in the section `host`? If this is the case, shouldn't we leave the two lines in the construction scripts rather than inserting a new dependency?

Reply:
I have just seen how this has been done in the recipe for "corrfunc". Good point, this simplifies the construction script. Thanks for the advice.
:+1: